database/src/main/java/org/akvo/flow/database/SurveyInstanceStatus.java

```diff
@@ -22,8 +22,8 @@
 
 public class SurveyInstanceStatus {
     public static final int SAVED = 0;
-    public static final int SUBMITTED = 1;
-    public static final int EXPORTED = 2;
+    public static final int SUBMIT_REQUESTED = 1;
+    public static final int SUBMITTED = 2;
```

Comment:
Can you maybe add comments here describing what each state represents?  That may be a helpful guide.

Reply:
Done!